spec/workers/notification_worker_spec.rb

```diff
@@ -71,6 +71,50 @@ def make_it_perform
       end
     end
 
+    context "filtering on document_type" do
+      context "when present in subscriber list" do
```

Comment:
Might be more clearly written as: "when the optional document_type is set on the subscriber list"

Or similar?